app/webpacker/src/stylesheets/_help_find_commodity.scss

```diff
@@ -0,0 +1,33 @@
+.call-to-action, .gem-c-govspeak {
```

Comment:
Do we really need all of this extra styling? Is there nothing built into the govuk libs that we can just use?